### PR TITLE
3520 - 0

### PR DESCRIPTION
**Pull request by SadTigger**
## Номер
3520

## Номер задания
0

## Ссылка на видео с демо
https://youtu.be/mKEgAc1vv-w

## Комментарии
All koans passed.

### CODE REVIEW

File: 3520/0/about_dice_project.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+# This class smells of :reek:InstanceVariableAssumption
+class DiceSet
+  attr_reader :values
+
+  def roll(set_size)
+    @values = []
+    set_size.times { @values.push rand(1..6) }
```
Comment (vsevolod) [inline 3520/0/about_dice_project.rb]: You need to do it through array initialization

File: 3520/0/about_java_interop.rb [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+# * Resolving module/class name conflicts
+# * Showing what gets returned
+# * Ruby Strings  VS Java Strings
+# * Calling custom java class
+# * Calling Ruby from java???
+
+class AboutJavaInterop < Neo::Koan
+  def test_using_a_java_library_class
+    java_array = java.util.ArrayList.new
+    assert_equal __, java_array.class
```
Comment (vsevolod) [inline 3520/0/about_java_interop.rb]: Why you didn't do that file?

File: 3520/0/about_variable_scope.rb [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    end
+
+    def Mouse.count
+      @@total
+    end
+  end
+
+  def test_instance_variable
+    oscar = Mouse.new('Oscar')
+    assert_equal __, oscar.name
```
Comment (vsevolod) [inline 3520/0/about_variable_scope.rb]: Complete this tasks too

File: 3520/0/about_java_interop.rb (hunk 2 above)
Comment (SadTigger) [reply 3520/0/about_java_interop.rb]: I use ruby 2.6 and that file are for using jruby. If I must install it and done this task - I'll do it.

File: 3520/0/about_variable_scope.rb (hunk 3 above)
Comment (SadTigger) [reply 3520/0/about_variable_scope.rb]: This task does'n include in path, but I'll find out how to complete it.

File: 3520/0/about_java_interop.rb (hunk 2 above)
Comment (vsevolod) [reply 3520/0/about_java_interop.rb]: Ok. You can miss it

File: 3520/0/about_java_interop.rb [hunk 4]
```diff
@@ -132,7 +132,7 @@ def test_java_class_are_open_from_ruby
     java_array = java.util.ArrayList.new
     java_array.add_all([1, 2, 3, 4, 5])
 
-    assert_equal __, java_array.multiply_all
+    assert_equal 120, java_array.multiply_all
```
Comment (vsevolod) [inline 3520/0/about_java_interop.rb]: Nice work :+1:

**Pull request merged**